funnel/static/translations/hi_IN/messages.json

```diff
@@ -1509,9 +1508,54 @@
       ],
       "Open now": [null, "अभी खोलें"],
       "Open later": [null, "बाद में खोलें"],
+      "This project is brought to you by": [null, ""],
```

Comment:
Where is this from? It seems like it's cut mid sentence.

Reply:
<img width="393" alt="Screen Shot 2021-02-12 at 10 08 15 AM" src="https://user-images.githubusercontent.com/11267213/107730723-227b1100-6d1a-11eb-8e75-7270e9f92baf.png">